apps/preview/next/pages/showcases/Combobox/ComboboxBasic.tsx

```diff
@@ -0,0 +1,288 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { type ChangeEvent, type FormEvent, type KeyboardEvent, useState, useRef, useId } from 'react';
+import { useDebounce } from 'react-use';
+import { offset } from '@floating-ui/react-dom';
+import {
+  SfInput,
+  SfIconExpandMore,
+  useDisclosure,
+  SfListItem,
+  SfLoaderCircular,
+  useTrapFocus,
+  useDropdown,
+  SfSwitch,
+} from '@storefront-ui/react';
+import classNames from 'classnames';
+
+export function DisableSwitch({
+  disabledState,
+  setDisabledState,
+}: {
+  disabledState: boolean;
+  setDisabledState: (disabledState: boolean) => void;
+}) {
+  return (
+    <div className="mt-40">
+      {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+      <label className="flex items-center">
+        <SfSwitch checked={disabledState} value="disabled" onChange={() => setDisabledState(!disabledState)} />
+        <span className="text-base ml-[10px] text-gray-900 cursor-pointer font-body">Disabled/Enabled</span>
+      </label>
+    </div>
+  );
+}
+
+type SelectOption = {
+  label: string;
+  value: string;
+};
+
+const options: SelectOption[] = [
+  {
+    label: 'Afghanistan',
+    value: 'afghanistan',
+  },
+  {
+    label: 'Bahamas',
+    value: 'bahamas',
+  },
+  {
+    label: 'Canada',
+    value: 'canada',
+  },
+  {
+    label: 'Denmark',
+    value: 'denmark',
+  },
+  {
+    label: 'Egypt',
+    value: 'egypt',
+  },
+];
+
+// eslint-disable-next-line no-promise-executor-return
```

Comment:
since its block maybe it would be better to place this rule for whole file on top of it, IMHO it will look better, wdyt?

Reply:
thx, done

Reply:
thank you